service/worker/stream.go

```diff
@@ -101,6 +116,11 @@ func (ps *PushStream) Start() error {
 		for {
 			select {
 			case <-gctx.Done():
+				// close all subscriber channels
+				close(subscriberResponseCh)
```

Comment:
response channel should be closed by stream ? as stream will be writing to that ?